crates/nu-command/src/filters/upsert.rs

```diff
@@ -168,8 +168,10 @@ fn upsert(
             for idx in 0..*val {
                 if let Some(v) = input.next() {
                     pre_elems.push(v);
+                } else if idx == 0 {
+                    return Err(ShellError::AccessEmptyContent(*span));
```

Comment:
```suggestion
```
Do we still need this bit? I think it's better to show `max: 0` than `empty content`